spec/lib/presenters/schema_org_spec.rb

```diff
@@ -407,8 +407,8 @@ def generate_structured_data(args)
     end
 
     def generate_org(details)
-      GovukSchemas::RandomExample.for_schema(frontend_schema: "organisation") do |org|
-        org.merge(details)
+      GovukSchemas::RandomExample.for_schema(frontend_schema: "organisation").tap do |org|
```

Comment:
What kind of failure did this cause? The only reason why this would fail is that the `details` are invalid, so it would be more reliable to fix that then to disable the test.

Reply:
It was for this: https://ci.integration.publishing.service.gov.uk/job/govuk_publishing_components/job/breadcrums-updated/5/console

Reply:
It looks like it's generating invalid content here. Maybe https://github.com/alphagov/govuk-content-schemas/pull/818 didn't make the base path optional for links nested within links?

`ordered_parent_organisations` is defined with `#/definitions/frontend_links_with_base_path`, which has a `links` field with `#/definitions/frontend_links_with_base_path` again:

You could loosen up this definition to make it all valid:

https://github.com/alphagov/govuk-content-schemas/blob/50cd03e7219787963799faffd2bbd5f0f5475ec2/dist/formats/organisation/frontend/schema.json#L831-L834